### DISCUSSION TITLE
Welcome to luxtronik Discussions!

### DISCUSSION THREAD

**Discussion by BenPru** [BenPru/luxtronik#271] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ldoellin** [comment]
Good day,

I'm using home assistant, and I'm really glad that this luxtronik integration is still maintained.
I'm looking for the opportunity to limit the power consumption, which was introduced in 3.90.1

**Comment by aaronlutin** [comment]
Hi luxtronic enthousiasts, 

I want to change my operation mode with a automasation. But it won't work can someone help me. 
I use this action but it doesn't work. 

action: water_heater.set_operation_mode
metadata: {}
data:
  operation_mode: STATE_PERFORMANCE (what should is fill in here it tried several things) 
target:
  entity_id: water_heater.luxtronik_310310_0787_domestic_water

Is this the correct way of doing this ? 

![image](https://github.com/user-attachments/assets/9f506e97-ccdc-4b8e-a708-bfef46ab24bd)

**Comment by aaronlutin** [comment]
I want to put it in party mode

**Reply by BenPru** [reply]
> I want to put it in party mode

The HA state for Luxtronik Party Mode is **performance**:
https://github.com/BenPru/luxtronik/blob/bbab400dc016a63362460bf69b9e9a9c375aaaac/custom_components/luxtronik/water_heater.py#L47

Example:
```
action: water_heater.set_operation_mode
metadata: {}
data:
  operation_mode: performance
target:
  entity_id: water_heater.luxtronik*_domestic_water

```
